ide/coqide/protocol/xmlprotocol.ml

```diff
@@ -389,13 +389,13 @@ end = struct
       | State         -> of_status
       | Option_state  -> of_option_state
       | Option_value  -> of_option_value
-      | Coq_info      -> of_coq_info
+      | Rocq_info      -> of_rocq_info
```

Comment:
I guess we don't care about the misalignment

Reply:
At least I don't. Line-alignement incurs a linear overhead when tweaking lines resulting in horrible diffs, and anyway I'm not convinced by the output personally.